draft-ietf-quic-version-negotiation.md

```diff
@@ -28,6 +28,9 @@ author:
     organization: Mozilla
     email: ekr@rtfm.com
 
+normative:
```

Comment:
Out of curiosity, does a syntax like `{{Section 6 of !RFC8999}}` not work?

Reply:
It didn't when I last touched that code.